install.sh

```diff
@@ -372,6 +372,38 @@ get_release_version() {
     info "Using ${VERSION_K3S} as release"
 }
 
+# --- get k3s-selinux version ---
+get_k3s_selinux_version() {
+    available_version="k3s-selinux-1.2-2.${rpm_target}.noarch.rpm"
+    if [ "${INSTALL_K3S_SKIP_DOWNLOAD}" == true ]; then
+        return
+    fi
+    info "Finding available k3s-selinux versions"
+    case $DOWNLOADER in
+        curl)
+            DOWNLOADER_OPTS="-s --connect-timeout 3 --retry 3"
+            ;;
+        wget)
+            DOWNLOADER_OPTS="-q -O - --connect-timeout=3 --tries=3"
+            ;;
```

Comment:
Busybox's wget doesn't support these flags. I'm not sure how portable the `timeout` builtin is, but if that's available, we might use that instead?

```console
/ # wget -q -O - --connect-timeout=3 --tries=3 https://www.google.com
wget: unrecognized option '--connect-timeout=3'
BusyBox v1.36.0 (2023-03-03 23:15:03 UTC) multi-call binary.

Usage: wget [-cqS] [--spider] [-O FILE] [-o LOGFILE] [--header STR]
	[--post-data STR | --post-file FILE] [-Y on/off]
	[--no-check-certificate] [-P DIR] [-U AGENT] [-T SEC] URL...

Retrieve files via HTTP or FTP

	--spider	Only check URL existence: $? is 0 if exists
	--header STR	Add STR (of form 'header: value') to headers
	--post-data STR	Send STR using POST method
	--post-file FILE	Send FILE using POST method
	--no-check-certificate	Don't validate the server's certificate
	-c		Continue retrieval of aborted transfer
	-q		Quiet
	-P DIR		Save to DIR (default .)
	-S    		Show server response
	-T SEC		Network read timeout is SEC seconds
	-O FILE		Save to FILE ('-' for stdout)
	-o LOGFILE	Log messages to FILE
	-U STR		Use STR for User-Agent header
	-Y on/off	Use proxy
/ # timeout -h
timeout: invalid option -- 'h'
BusyBox v1.36.0 (2023-03-03 23:15:03 UTC) multi-call binary.

Usage: timeout [-s SIG] [-k KILL_SECS] SECS PROG ARGS

Run PROG. Send SIG to it if it is not gone in SECS seconds.
Default SIG: TERM.If it still exists in KILL_SECS seconds, send KILL.
```